apps/web/README.md

```diff
@@ -0,0 +1,61 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Launch `FluentUI Web Tester` app
+
+1. Make sure you have followed the Getting Started instructions [here](../../README.md) to install packages and build the entire repository.
+
+2. Then go into `apps\web` folder:
+
+```
+    cd apps/web
+```
+
+4. Launch the FluentUI Web Tester app:
+
+```
+    yarn web
+```
+5. Go to `localhost:8080` in your browser of choice and you will see the FluentUI Checkbox component show up.
+
+## Available Scripts
+
+In the project directory, you can run:
```

Comment:
Is this part of the README still valid, I thought these scripts didn't work anymore?